<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Controls</class>
 <widget class="QWidget" name="Controls">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>976</width>
    <height>157</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="50,50">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QDial" name="dialVolume">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="wrapping">
        <bool>false</bool>
       </property>
       <property name="notchesVisible">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="horizontalSliderPosition">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QScrollBar::groove:horizontal {
     border: 0px solid #999999;
     height: 1px; 
     background: gray;/*qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #B1B1B1, stop:1 #c4c4c4);*/
     margin: 1px 5;
	border-radius: 1px;
 }

 QScrollBar::handle:horizontal {
     background: qlineargradient(x1:0, y1:0, x2:1, y2:1, stop:0 #b4b4b4, stop:1 #8f8f8f);
     border: 0px solid #5c5c5c;
     width: 10px;
     height: 10px;
     margin: -5px; 
     border-radius: 5px;
 }</string>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLCDNumber" name="lcdNumberElapsed">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <property name="segmentStyle">
          <enum>QLCDNumber::Flat</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <pointsize>16</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLCDNumber" name="lcdNumberDuration">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <property name="segmentStyle">
          <enum>QLCDNumber::Flat</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="sizeConstraint">
        <enum>QLayout::SetMinimumSize</enum>
       </property>
       <item>
        <widget class="QPushButton" name="pushButtonPlayPause">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>play/pause</string>
         </property>
         <property name="iconSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonPrev">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>prev</string>
         </property>
         <property name="iconSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonNext">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>next</string>
         </property>
         <property name="iconSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>dialVolume</sender>
   <signal>valueChanged(int)</signal>
   <receiver>Controls</receiver>
   <slot>onVolumeChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>58</x>
     <y>43</y>
    </hint>
    <hint type="destinationlabel">
     <x>5</x>
     <y>52</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonNext</sender>
   <signal>clicked()</signal>
   <receiver>Controls</receiver>
   <slot>onNextClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>731</x>
     <y>42</y>
    </hint>
    <hint type="destinationlabel">
     <x>568</x>
     <y>52</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonPrev</sender>
   <signal>clicked()</signal>
   <receiver>Controls</receiver>
   <slot>onPrevClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>640</x>
     <y>52</y>
    </hint>
    <hint type="destinationlabel">
     <x>736</x>
     <y>35</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonPlayPause</sender>
   <signal>clicked()</signal>
   <receiver>Controls</receiver>
   <slot>onPlayPauseClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>458</x>
     <y>70</y>
    </hint>
    <hint type="destinationlabel">
     <x>464</x>
     <y>95</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onVolumeChanged(int)</slot>
  <slot>onPlayPauseClicked()</slot>
  <slot>onStopClicked()</slot>
  <slot>onNextClicked()</slot>
  <slot>onPrevClicked()</slot>
 </slots>
</ui>
